pkg/kubelet/server.go

```diff
@@ -60,7 +60,7 @@ type TLSOptions struct {
 
 // ListenAndServeKubeletServer initializes a server to respond to HTTP network requests on the Kubelet.
 func ListenAndServeKubeletServer(host HostInterface, address net.IP, port uint, tlsOptions *TLSOptions, enableDebuggingHandlers bool) {
-	glog.V(1).Infof("Starting to listen on %s:%d", address, port)
+	glog.Infof("Starting to listen on %s:%d", address, port)
```

Comment:
I don't see vlog being set everywhere. For example hack/test-cmd.sh does not set vlog.